lib/include/hse_ikvdb/kvdb_meta.h

```diff
@@ -89,6 +89,19 @@ kvdb_meta_deserialize(struct kvdb_meta *meta, const char *kvdb_home);
 merr_t
 kvdb_meta_usage(const char *kvdb_home, uint64_t *size);
 
+/**
+ * Upgrade KVDB meta
+ *
+ * @param meta: KVDB metadata
+ * @param kvdb_home: KVDB home
+ * @returns Error status
+ * @retval 0 on succes
+ * @retval !0 on error
+ */
+/* MTF_MOCK */
+merr_t
+kvdb_meta_upgrade(struct kvdb_meta *const meta, const char *const kvdb_home);
```

Comment:
Trying to understand this function and how it is used. Say I have a component like the mblock header which is out of date for my KVDB. The global omf version for my KVDB is 10, but the newest is 11. What is the order of events in terms of getting my KVDB up to global omf version 11? I am assuming there is more than just writing an 11 and the upgraded mblock header version.

Reply:
> The function kvdb_meta_upgrade() updates the 'omf_version' version field after all components have successfully initialized during ikvdb_open().

This was what I wanted to double check. Thanks.